src/pages/home/TrendingSection.tsx

```diff
@@ -62,21 +73,15 @@ export default function TrendingSection() {
                 />
               ))}
             </Grid>
-          </Space>
 
-          <Row
-            justify="center"
-            style={{
-              padding: '1rem 0 2rem 0',
-            }}
-          >
             <Link href="/projects">
               <Button size="large" block={isMobile}>
                 <Trans>More trending projects</Trans>
```

Comment:
<img width="500" alt="Screen Shot 2022-11-18 at 4 45 27 pm" src="https://user-images.githubusercontent.com/96150256/202638628-19f44e4b-424f-4f29-a75b-9298c0118f7e.png">

**Nit:** This one doesn't have a link showing up on hover